nuxeo-filter.html

```diff
@@ -281,6 +281,25 @@
       }
 
       return this._getBinding(prop, dataHost.dataHost);
-    }
+    },
+
+      _addNuxeoConnectionElement: function(model) {
+          var nuxeoConnection = document.createElement('nuxeo-connection');
+          this.user = nuxeoConnection.user;
+      },
+
+      _setNuxeoConnectionListener: function(){
+          window.addEventListener('nuxeo-connected', function (e) {
+              this.user = e.detail.user;
+          }.bind(this));
+      },
+
+      ready: function () {
+          if (!this.user) {
+              //If user is not provided, add nuxeo-connection element and get user from it.
+            this._addNuxeoConnectionElement();
+            this._setNuxeoConnectionListener();
```

Comment:
We probably don't need these as private methods since they are pretty small and are not used anywhere else.